solutions/gold/cses-1716.mdx

```diff
@@ -68,9 +68,7 @@ ll binpow(ll x, ll n, ll m) {
 	return res;
 }
 
-ll binom(int n, int k) {
-	return fac[n] * inv[k] % MOD * inv[n - k] % MOD;
-}
+ll binom(int n, int k) { return fac[n] * inv[k] % MOD * inv[n - k] % MOD; }
```

Comment:
bruh.

Reply:
lol guess we're having a code convention that's being enforced automatically